### ISSUE TITLE
ios landscape device rotates image so that the floor is to the right 

### ISSUE THREAD

**Issue by breezewaylogic** [hjam40/Camera.MAUI#101]
ios landscape pictures do not work.  The cameraview rotates to image too so that the floor is to the right.

**Comment by ELSayedpaxera** [comment]
is there is any update ..??

**Comment by ELSayedpaxera** [comment]
is there is any update  ... ?

**Comment by breezewaylogic** [comment]
Yes.   Easy fix.  The IOS configuration in the project must be set to allow for landscape mode.

**Comment by victorbvir2al** [comment]
@breezewaylogic  What you mention is only for the app, but the camera keeps rotating
